hw02-collections/src/main/java/org/otus/homework/CustomerService.java

```diff
@@ -0,0 +1,30 @@
+package org.otus.homework;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.TreeMap;
+
+public class CustomerService {
+    private final Map<Customer, String> customers = new TreeMap<>();
```

Comment:
NavigableMap
```suggestion
    private final NavigableMap<Customer, String> customers = new TreeMap<>();
```
И тогда с `getSmallest()` и `getNext()` станет понятнее, что делать.